.gitignore

```diff
@@ -68,3 +69,5 @@ nbproject/
 modules.xml
 .idea/misc.xml
 *.ipr
+**/.vscode/*
+c.tmp
```

Comment:
what is `c.tmp`?

Reply:
vscode generates it in the root folder

Reply:
I have never see that while working with vscode... Normally vscode should write all its files to .vscode (or somewhere in your home directory)

If you are absolutely sure that this file has to be gitignored and it wasn't just a one time thing, please add a comment saying what `c.tmp` is and why it is ignored.

Also the suggested gitignore for vscode is:
```gitignore
.vscode/*
!.vscode/settings.json
!.vscode/tasks.json
!.vscode/launch.json
!.vscode/extensions.json
```
https://github.com/github/gitignore/blob/master/Global/VisualStudioCode.gitignore